src/resources/assets/js/components/enso/bulma/BooleanFilter.vue

```diff
@@ -15,25 +16,17 @@ import VueFilter from '../bulma/VueFilter.vue';
 
 export default {
     components: { VueFilter },
-    props: {
-        yesLabel: {
-            type: String,
-            default: 'Yes',
-        },
-        noLabel: {
-            type: String,
-            default: 'No',
-        },
-    },
 
```

Comment:
... we need to declare the only the `value` prop, that is passed in the template
```
props: {
     value: {
          type: null,
          default: null,
      },
},
```
(.)